apps/storybook/src/Table.stories.tsx

```diff
@@ -314,9 +314,15 @@ export const Sortable: Story<Partial<TableProps>> = (args) => {
         columns: [
           {
             id: 'name',
-            Header: 'Name',
+            Header: 'Name (sorts ascending first)',
             accessor: 'name',
           },
+          {
+            id: 'id',
```

Comment:
nit, but having `id` after `name` looks uncommon.
id -> name -> description -> actions

But I noticed now in other stories we have even different, where `id` goes after `description` :/
Leave it, I guess...

Reply:
Yeah, it seemed weird to have it after description, and I wanted it to be adjacent to the asc first column. But I did change it to your example.